wiki/markdown/usage/service.md

```diff
@@ -204,9 +204,12 @@ https://localhost:8080/eventrepository/events/?meta.type=EiffelArtifactCreatedEv
                 }]
 ```
 
-#### Lookups are provided with two options:
+#### Lookups are provided with four options:
 
 | Options              | Default Value | Description                                                                                                                                                                                                                                                                                                                                                                                                                            |
 |----------------------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | failIfMultipleFound: | False         | If value is set to false, REMReM generates event if multiple event ids found or no event id fetched from the event repository configured in generate and adds them in links section of the target field. If value is set to true, if only one event is found then REMReM generates the event successfully and if multiple event ids are found then REMReM fails to generate any event and displays relevant error message to the user. |
```

Comment:
This description is long and hard to understand fully. Could you write like this instead?:

> If value is set to True and multiple event ids are found through any of the provided lookup definitions, then no event will be generated.

Reply:
Done